Packs/TrustwaveFusion/Integrations/TrustwaveFusion/TrustwaveFusion.py

```diff
@@ -0,0 +1,587 @@
+    If the input is None, it will throw a ValueError if required is ``True``,
+    or ``None`` if required is ``False.
+
+    :type arg: ``Any``
+    :param arg: argument to convert
+
+    :type arg_name: ``str``
+    :param arg_name: argument name
+
+    :type required: ``bool``
+    :param required:
+        throws exception if ``True`` and argument provided is None
+
+    :return:
+        returns an ``int`` containing a timestamp (seconds from epoch) if conversion works
+        returns ``None`` if arg is ``None`` and required is set to ``False``
+        otherwise throws an Exception
+    :rtype: ``Optional[int]``
+    """
+
+    if arg is None:
+        if required is True:
+            raise ValueError(f'Missing "{arg_name}"')
+        return None
+
+    if isinstance(arg, str) and arg.isdigit():
+        # timestamp is a str containing digits - we just convert it to int
+        return int(arg)
+    if isinstance(arg, str):
+        # we use dateparser to handle strings either in ISO8601 format, or
+        # relative time stamps.
+        # For example: format 2019-10-23T00:00:00 or "3 days", etc
+        date = dateparser.parse(arg, settings={"TIMEZONE": "UTC"})
+        if date is None:
+            # if d is None it means dateparser failed to parse it
+            raise ValueError(f"Invalid date: {arg_name}")
+
+        return int(date.replace(tzinfo=timezone.utc).timestamp())
+    if isinstance(arg, (int, float)):
+        # Convert to int if the input is a float
+        return int(arg)
+    raise ValueError(f'Invalid date: "{arg_name}"')
+
+
+def format_notes(notes, limit=5):
+    """Format notes json from the Fusion API"""
+    results = []
+    for note in notes:
+        actor = note.get("actor", "")
+        ts = note.get("timestamp", "")
+        text = note.get("text", "")
+        results.append(f"{ts} Created by: {actor}")
+        results.append("NOTE:")
+        for line in text.split("\n"):
+            results.append(f"{line}")
+        results.append("----------------")
+    if results:
+        results.pop()
+    return "\n".join(results)
+
+
+''' COMMAND FUNCTIONS '''
+
+
+def test_module(client: Client) -> str:
+    """Tests API connectivity and authentication'
+
+    Returning 'ok' indicates that the integration works like it is supposed to.
+    Connection to the service is successful.
+    Raises exceptions if something goes wrong.
+
+    :type client: ``Client``
+    :param Client: client to use
+
+    :return: 'ok' if test passed, anything else will fail the test.
+    :rtype: ``str``
+    """
+
+    message: str = ""
+    try:
+        client.describe()
+        message = "ok"
+    except DemistoException as e:
+        if e.res is not None and e.res.status_code == 401:
+            message = "Authorization Error: make sure API Key is correctly set"
+        elif "Forbidden" in str(e) or "Authorization" in str(e):
+            message = "Authorization Error: make sure API Key is correctly set"
```

Comment:
its the same thing, why the need to check it individually, one of them is enough